haxr.cabal

```diff
@@ -1,5 +1,5 @@
 Name: haxr
-Version: 3000.11.1
+Version: 3000.11.2
```

Comment:
Also, please don't bump the version number, I will bump it appropriately in a separate commit.